From 2d19fa7c19cab33e1adcbe099c801e2569943b4d Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <nathan@kernel.org>
Date: Fri, 31 Mar 2023 08:42:37 -0700
Subject: [PATCH 02/11] NOTCBL: WIP: cdx: Mark bus_type parameter as const in
 rescan_store()

Commit 75cff725d956 ("driver core: bus: mark the struct bus_type for
sysfs callbacks as constant") interacting with commit 2959ab247061
("cdx: add the cdx bus driver").

Signed-off-by: Nathan Chancellor <nathan@kernel.org>
---
 drivers/cdx/cdx.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/cdx/cdx.c b/drivers/cdx/cdx.c
index 67c32cb2c006..38511fd36325 100644
--- a/drivers/cdx/cdx.c
+++ b/drivers/cdx/cdx.c
@@ -363,7 +363,7 @@ static struct attribute *cdx_dev_attrs[] = {
 };
 ATTRIBUTE_GROUPS(cdx_dev);
 
-static ssize_t rescan_store(struct bus_type *bus,
+static ssize_t rescan_store(const struct bus_type *bus,
 			    const char *buf, size_t count)
 {
 	struct cdx_controller *cdx;
-- 
2.40.0

